pkg/api/pod/util.go

```diff
@@ -284,6 +286,24 @@ func DropDisabledRunAsGroupField(podSpec *api.PodSpec) {
 	}
 }
 
+// DropDisabledProcMountField removes disabled fields from PodSpec related
+// to ProcMount
+func DropDisabledProcMountField(podSpec *api.PodSpec) {
```

Comment:
I don't see the equivalent for DropDisabledAllowedProcMountTypes on PSP - am I just blind?

Reply:
fixed :)